src/bmm.jl

```diff
@@ -51,7 +51,7 @@ function bmm!(transA::AbstractChar, transB::AbstractChar, alpha::Number, A::Knet
         n=kb; k==nb;
     end
     (m == size(C,1) && n == size(C,2) && bs == size(C,3)) || throw(DimensionMismatch("$(map(size,(A,B,C)))"))
-    lda,ldb,ldc=ma,ka,ma
+    lda,ldb,ldc=ma,kb,ma
```

Comment:
> Do you want to convert some of the things from [this]?(https://github.com/ekinakyurek/KnetLayers.jl/blob/v0.2.0-examples/src/batchedmul.jl)
> 
> It is writen on same gpu function, but also has faster cpu version from Transformers.jl + input transposition support.

Sure, but I cannot change the interface, so some functions would have to be renamed (to bmm and bmm!). Do you have a speed benchmark?

Reply:
> Did you try for all cases of transA, transB?
> 
> because below worked for me and different than yours?
> 
> ```julia
> lda,ldb,ldc= ma,kb,m
> ```

Yes, I just added tests that at least show the operation goes through without error for all combinations of 'N' and 'T'. Do you have an example that shows a bug?